docs/get-started/try-viam/rover-resources/rover-tutorial-2.md

```diff
@@ -0,0 +1,383 @@
+
+Your voltage cutoff circuit is now configured.
+When the voltage drops below 14.8V (or reaches the cutoff point you chose), a relay will disconnect the motherboard.
+A minimum voltage of 14.9V will be needed to reconnect the power (that is, after charging the batteries).
+
+### Connect the ribbon cable, Pi, and camera
+
+{{% alert title="Tip" color="tip" %}}
+If you are using another board, follow the instructions in [Alternative board configurations](#alternative-board-configurations) in place of this step.
+{{% /alert %}}
+
+To be able to attach the Raspberry Pi, unscrew the top of the rover with the biggest Allen key.
+Then use the smallest Allen key and the provided M2.5 screws to attach the Raspberry Pi to your rover through the standoffs on the motherboard.
+The Raspberry Pi 4 should be mounted such that the USB ports are to the right, as viewed from above.
+
+Use the ribbon cable to connect the Raspberry Pi 4 to the motherboard.
+The ribbon cable comes connected to the motherboard out of the box, wrap it over the top of the Raspberry Pi 4 and connect with the GPIO pins as shown:
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/ribbon-cable1.png" resize="400x" declaredimensions=true alt="Ribbon cable attached to pi" >}}
+<br>
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/ribbon-cable2.png" resize="400x" declaredimensions=true alt="Ribbon cable attached to pi" >}}
+
+Also, connect the webcam's USB lead to any USB port on your Pi.
+
+Assuming you are using a Raspberry Pi 4, you can skip the following section and move to [Screw the top plate back on and switch the rover on](#screw-the-top-plate-back-on-and-switch-the-rover-on).
+
+#### Alternative board configurations
+
+This guide assumes you are using a Raspberry Pi 4, but you can use different [boards](/components/board/) with your Viam Rover 2 with some modifications while attaching the board.
+
+If you are using one of these boards, reference the appropriate alternative hole patterning provided on the motherboard:
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/hole-patterning.png" resize="400x" declaredimensions=true alt="Viam rover 2 motherboard hole patterns" >}}
+
+Disattach the motherboard, unscrew the standoffs, and move them to the correct holes.
+Then, use the smallest Allen key and the provided M2.5 screws to attach your board to your rover through these standoffs.
+If you are using a Jetson board, you will have to purchase appropriately sized standoffs.
+
+If you are using a Raspberry Pi Zero 2W, you should be able to connect your ribbon cable straight to the board.
+If not, you will have to take off the ribbon cable and use [dupont connectors](https://www.amazon.com/IWISS-1550PCS-Connector-Headers-Balancer/dp/B08X6C7PZM/) to wire a connection from the motherboard to the single-board computer's GPIO pins.
+
+Also, connect the webcam's USB lead to any USB port on your board.
```

Comment:
Since you're not connecting anything else to a usb port this feels better: 
```suggestion
Then connect the webcam's USB lead to any USB port on your board.
```